src/main/java/org/sunbird/digilocker/model/PullDocResponse.java

```diff
@@ -0,0 +1,121 @@
+package org.sunbird.digilocker.model;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+@JacksonXmlRootElement(localName = "PullDocResponse", namespace = "http://tempuri.org/")
+public class PullDocResponse {
+    private ResponseStatus responseStatus;
+    private DocDetails docDetails;
+
+    @JacksonXmlProperty(localName = "ResponseStatus", namespace = "http://tempuri.org/")
+    public ResponseStatus getResponseStatus() {
+        return responseStatus;
+    }
+
+    public void setResponseStatus(ResponseStatus responseStatus) {
+        this.responseStatus = responseStatus;
+    }
+
+    @JacksonXmlProperty(localName = "DocDetails", namespace = "http://tempuri.org/")
+    public DocDetails getDocDetails() {
+        return docDetails;
+    }
+
+    public void setDocDetails(DocDetails docDetails) {
+        this.docDetails = docDetails;
+    }
+
+    public static class ResponseStatus {
```

Comment:
I use inner class because of mapping the xml format i can create different class but their are 6 object which i check for which need to create sperate class